### PR TITLE
hwdef: added files for Airvolute DCS2 onboard FMU

### PR DESCRIPTION
**Pull request by vrsanskytom**
- added hwdef files for Airvolute DCS2 onboard FMU

Current state: 
-Bl: working
-FW: running, possible to connect through MAVLINK and onboard USB connection

### CODE REVIEW

**Comment by vrsanskytom** [comment]
Added some default parameters. Device tested and all of the features works properly.

**Comment by Ryanf55** [comment]
MacOS failures are unrelated. I performed a rebase.

File: libraries/AP_HAL_ChibiOS/hwdef/Airvolute-DCS2/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+ - 4 PWM outputs 
+ - Buzzer output
+ - USB connection onboard with Jetson Host
+
+## UART Mapping
+
+ - SERIAL0 -> USB
+ - SERIAL1 -> UART2 (Telem1)
+ - SERIAL2 -> UART3 (Telem2)
+
```
Comment (Ryanf55) [inline libraries/AP_HAL_ChibiOS/hwdef/Airvolute-DCS2/README.md:22]: Does it only have 3 UARTs?

**Comment by tridge** [comment]
marked for doc review by Henry - needs pinout information in the README

File: libraries/AP_HAL_ChibiOS/hwdef/Airvolute-DCS2/README.md (hunk 1 above)
Comment (Hwurzburg) [reply libraries/AP_HAL_ChibiOS/hwdef/Airvolute-DCS2/README.md:22]: and one used as RC....

**Comment by Hwurzburg** [comment]
@vrsanskytom can you send me the wiring diagram so I can review? I will need that and a pinout list for the board to review for wiki page...and other suggestions, like default protocols,etc.

**Comment by vrsanskytom** [comment]
@Hwurzburg OK, we will update wiki today and try to make it containing everything important.

**Comment by Hwurzburg** [comment]
making a wiki page like : https://ardupilot.org/copter/docs/common-holybro-pixhawk6X.html is certainly appreciated, but I usually make that page based off the README.md file here. Its important that it be accurate and complete not only to assist in the wiki page production, but also as a code document for other developer's reference. I check the README.md versus the hwdef for consistency and completeness, and check the wiring diagram and pinouts like a user to be sure nothing is confusing...for example, you specify UART3 as the only RC input but do not default its protocol to RCin which would be expected by a user (also since its only RX pinned out,it wont support RC like CRSF,ELRS, or SRXL2)...

**Comment by vrsanskytom** [comment]
@Hwurzburg  We just updated the readme, now it should contain all of the important information. 

I`m not sure if I understand properly what does it mean to remove hwdef line. But the whole board definitely is offered commercially, and we would like users to have the possibility to upgrade the firmware (firmware updates are handled from companion computer).

File: libraries/AP_HAL_ChibiOS/hwdef/Airvolute-DCS2/hwdef.dat [hunk 2]
```diff
@@ -0,0 +1,176 @@
+# hw definition file for processing by chibios_hwdef.py
+#following exapmle: 
+#https://github.com/ArduPilot/ardupilot/blob/master/libraries/AP_HAL_ChibiOS/hwdef/mRoControlZeroH7/hwdef.dat#L105
+# for H743 bootloader
+
+AUTOBUILD_TARGETS None
```
Comment (Hwurzburg) [inline libraries/AP_HAL_ChibiOS/hwdef/Airvolute-DCS2/hwdef.dat:6]: delete this line or it wont be autobuild on the the firmware server for all vehicle types

**Review by Hwurzburg — CHANGES_REQUESTED** [review]

**Review by Hwurzburg — CHANGES_REQUESTED** [review]
a few suggested changes

**Comment by vrsanskytom** [comment]
Thanks for review. To what parts do you need to split it ?

**Review by Hwurzburg — APPROVED** [review]

**Review by tridge — APPROVED** [review]

**Pull request merged**